alf/bin/train_play_test.py

```diff
@@ -124,6 +124,9 @@ def _to_conf_params(parameters):
     'TrainerConfig.num_checkpoints=1',
     # disable evaluate
     'TrainerConfig.evaluate=False',
+    # Do not get stuck there and asking for confirmation upon crash.
+    # Die and give feedback immediately for unit tests.
+    'confirm_checkpoint_upon_crash=False',
```

Comment:
Adding this because otherwise running the unit tests that are supposed to crash immediately will hang forever locally.